doc/api/http.md

```diff
@@ -79,7 +92,11 @@ http.get(options, (res) => {
 });
 ```
 
-Alternatively, you could just opt out of pooling entirely using
+You may also use an agent for an individual request. By providing
+`{agent: false}` as an option to the `http.get()` or `http.request()`
+functions, a one-time use `Agent` with default options and no connection
```

Comment:
Is this really true? An Agent is allocated even if it doesn't need one, as opposed to just making a direct HTTP request with no agent at all? I'm surprised, so if true, its good its documented.